test/System/Viewport/CanCreateViewportOnSheet.dyn

```diff
@@ -0,0 +1,333 @@
+{
+  "Uuid": "d9477586-5b5c-425b-8699-406fac94ad32",
+  "IsCustomNode": false,
+  "Description": null,
+  "Name": "CanCreateViewportOnSheet",
+  "ElementResolver": {
+    "ResolutionMap": {
+      "Revit.Elements.ElementSelector": {
+        "Key": "Revit.Elements.ElementSelector",
+        "Value": "RevitNodes.dll"
+      }
+    }
+  },
+  "Inputs": [],
+  "Outputs": [],
+  "Nodes": [
+    {
+      "ConcreteType": "Dynamo.Graph.Nodes.ZeroTouch.DSFunction, DynamoCore",
+      "NodeType": "FunctionNode",
+      "FunctionSignature": "Revit.Elements.Viewport.Create@Revit.Elements.Views.Sheet,Revit.Elements.Views.View,Autodesk.DesignScript.Geometry.Point",
```

Comment:
This node should be Viewport.BySheetViewLocation , there is no such Viewport.Create node in your code.